mdx2/command_line/merge.py

```
@@ -2,45 +2,48 @@
 
 
-def parse_arguments():
-    """Parse commandline arguments"""
+@dataclass
+class Parameters:
+    """Options for merging scaled intensities"""
 
-    parser = argparse.ArgumentParser(
-        description=__doc__,
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    hkl: List[str] = field(positional=True, nargs="+")  # NeXus file(s) containing hkl_table
```

Ruff (0.14.8)

Do not perform function call `field` in dataclass defaults

(RUF009)